README.md

```diff
@@ -21,8 +21,9 @@ If you haven’t yet installed [`rads`](https://github.com/PHSKC-APDE/rads), fol
 
 ## Getting started
 After installation, we highly recommend that you start by walking through a vignette on the [wiki](https://github.com/PHSKC-APDE/rads/wiki).
-* [Get Population Data](https://github.com/PHSKC-APDE/rads/wiki/Vignette:-get_population())
-* [Exploring Birth Data](https://github.com/PHSKC-APDE/rads/wiki/Vignette:-get_data_birth())
+* [calc()](https://github.com/PHSKC-APDE/rads/wiki/Vignette:-calc())
+* [get_population()](https://github.com/PHSKC-APDE/rads/wiki/Vignette:-get_population())
```

Comment:
Can we add the following?
* [age_standardize()](https://github.com/PHSKC-APDE/rads/wiki/Vignette:-age_standardize())